paymentsheet/src/test/java/com/stripe/android/paymentsheet/state/DefaultAnalyticsMetadataFactoryTest.kt

```diff
@@ -674,6 +675,78 @@ class DefaultAnalyticsMetadataFactoryTest {
         assertThat(appearance?.get("usage")).isEqualTo(true)
     }
 
+    @Test
+    fun `create returns true for link_native_available when native link is available`() = runScenario {
+        val fakeLinkGate = FakeLinkGate().apply {
+            setUseNativeLink(true)
+        }
+        val linkGateFactory = FakeLinkGate.Factory(fakeLinkGate)
+
+        val resultMap = createAnalyticsMetadata(
+            linkGateFactory = linkGateFactory
+        )
+
+        assertThat(resultMap["link_native_available"]).isEqualTo(true)
+    }
+
+    @Test
+    fun `create returns false for link_native_available when native link is not available`() = runScenario {
```

Comment:
Are these tests all that useful? All they're really testing is the functionality of the fake link gate?

I think having a single test with a true value is probably enough, right?